docs/development/roadmap.md

```diff
@@ -7,27 +7,29 @@ The following improvements are actively being worked on:
 
 - https://github.com/gchq/sleeper/issues/3446 Bulk export.
 - https://github.com/gchq/sleeper/issues/1330 Support deploying a published version of Sleeper.
-- https://github.com/gchq/sleeper/issues/1389 Upgrade to AWS SDK v2.
+- https://github.com/gchq/sleeper/issues/5078 Improvements to bulk import.
+- https://github.com/gchq/sleeper/issues/4344 Aggregation support for compactions in Rust with DataFusion.
 
 The following are likely to be worked on in the near future:
 
+- https://github.com/gchq/sleeper/issues/5083 Queries in Rust with DataFusion.
 - https://github.com/gchq/sleeper/issues/4393 Batch up partition splitting commits.
 - https://github.com/gchq/sleeper/issues/3693 Improvements to declarative deployment with infrastructure as code.
 - https://github.com/gchq/sleeper/issues/1391 Create a library of repeatable, sustained, large-scale performance tests.
-- https://github.com/gchq/sleeper/issues/1388 Rust implementations for operations on data files.
 
 The following improvements will be worked on in future (these are in no particular order):
 
+- https://github.com/gchq/sleeper/issues/1388 Other Rust implementations for operations on data files.
 - https://github.com/gchq/sleeper/issues/576 Use Arrow types in the table schema.
 - https://github.com/gchq/sleeper/issues/4396 Failure handling / backpressure for state store updates.
 - https://github.com/gchq/sleeper/issues/4398 Trigger compaction dispatch in transaction log follower.
+- https://github.com/gchq/sleeper/issues/4555 Optimise table snapshot creation.
 - Scaling improvements.
     - https://github.com/gchq/sleeper/issues/4525 Mitigate limitations on throughput of state store updates.
     - https://github.com/gchq/sleeper/issues/4215 Service that maintains an up-to-date cache of the state store.
     - https://github.com/gchq/sleeper/issues/4218 Batch up updates to job trackers from state store commits.
     - https://github.com/gchq/sleeper/issues/4214 Mitigate memory limitations with multiple Sleeper tables.
     - https://github.com/gchq/sleeper/issues/4395 Table state partitioning.
```

Comment:
Can we delete the table state partitioning line? That's part of a bigger epic about improving the state store, and we don't know yet if we intend to improve that using table state partitioning or other approaches. I know it wasn't added in this PR, but can we tidy it at the same time, please?

Reply:
Yes, this is included under the first item on this list, to mitigate limitations on the throughput of the state store. I'll remove it.